avocado-core/src/main/scala/org/bdgenomics/avocado/algorithms/math/LogPhred.scala

```diff
@@ -0,0 +1,29 @@
+package org.bdgenomics.avocado.algorithms.math
```

Comment:
I think that script is only in the ADAM repository. Can move/run the same script?